Code/GraphMol/new_canon.cpp

```diff
@@ -170,7 +170,6 @@ void compareRingAtomsConcerningNumNeighbors(Canon::canon_atom *atoms,
     memset(lastLevelNbrs.get(), 0, nAtoms * sizeof(char));
     memset(currentLevelNbrs.get(), 0, nAtoms * sizeof(char));
     memset(revisitedNeighbors.get(), 0, nAtoms * sizeof(int));
-    unsigned count = 1;
```

Comment:
Did you mean for this to be included here?

Reply:
[-Werror,-Wunused-but-set-variable]


Is this the new g++11 default?

Reply:
The errors are from clang-16. I've addressed them later in the PR

Reply:
Of course it's clang (OSX and all)